subroutines.sh

```diff
@@ -85,9 +99,26 @@ bluetoothctl_read() {
   # Read BLE data from bluetoothctl or an input file
   if [ -z $BLECTL_FILE_INPUT ]; then
     log_debug "Launching bluetoothctl to check for BLE presence"
-    BLECTL_TIMEOUT=11
     set +e
-    BLTCTL_OUT=$(bluetoothctl --timeout $BLECTL_TIMEOUT scan on 2>&1 | grep -v DEL)
+    BLTCTL_OUT=$({
+      if [ $BLTCTL_COMMAND_DEVICES == "true" ]; then
+        bltctlCommands="power on,devices,scan on"
```

Comment:
power on will not cause issues if the bluetooth device is used by something else?

Reply:
Well... I had to do that for the `scan on` to work on mine. As you can see below, I kept the output cuz it was interesting... I sent the command `devices` and nothing showed up. Did `power on` and then another `devices` command.

```
[bluetooth]# hci0 new_settings: powered bondable ssp br/edr le secure-conn wide-band-speech
[bluetooth]# [CHG] Controller 04:42:1A:5B:2B:00 Pairable: yes
[bluetooth]# AdvertisementMonitor path registered
[bluetooth]# devices
[bluetooth]# power on
[bluetooth]# Changing power on succeeded
[bluetooth]# devices
hci0 type 6 discovering on
[CHG] Controller 04:42:1A:5B:2B:00 Discovering: yes
[NEW] Device 71:D4:D8:AB:FC:A2 71-D4-D8-AB-FC-A2
[NEW] Device 70:72:8E:07:4C:4E 70-72-8E-07-4C-4E
[NEW] Device 61:6A:54:6A:87:89 61-6A-54-6A-87-89
[NEW] Device 5B:62:F4:F9:03:BA 5B-62-F4-F9-03-BA
[NEW] Device 43:8A:B2:DC:87:DE 43-8A-B2-DC-87-DE
[NEW] Device 61:11:DD:7E:0E:45 61-11-DD-7E-0E-45
[NEW] Device 1C:B3:C9:11:77:6C 1C-B3-C9-11-77-6C
[NEW] Device 5D:93:2D:44:4F:D9 5D-93-2D-44-4F-D9
```

If the adapter is powered off, the power on is required. If a light bulb is already on, it won't blow up to "turn it on" again. If the adapter is already powered on, I doubt sending a "power on" has any impact.